source/app/controllers/sessions_controller.rb

```diff
@@ -1,12 +1,28 @@
 class SessionsController < ApplicationController
 
   def index
+    redirect_to current_user if logged_in?
   end
 
   def create
+    user_name = params[:session][:user_name]
+    password  = params[:session][:password]
+    user = User.find_by(user_name: user_name)
+
+    if user && user.authenticate(password)
+      log_in user
+      redirect_to user
+    else
+      redirect_to login_path
```

Comment:
Maybe show the user a message to explain why they are looking at the login form again?